.devcontainer/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM node:24-slim
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME/bin:$PATH"
+RUN corepack enable
+WORKDIR /app
+RUN apt update && apt install -y git curl gh
```

Hadolint (2.15.1)

[info] Delete the apt lists (/var/lib/apt/lists) after installing something

(DL3009)